.travis.yml

```diff
@@ -1,7 +1,6 @@
-env:
-  global:
-  - DEBUG='* -superagent -express:* -mocha:*'
 language: node_js
+sudo: false
```

Comment:
With this option travis might be sane again... it allows travis to use containers which is much faster... Still not sure if the networking is reliable, the main reason I've moving things to circle ci...